nbgitpuller/plugins/googlePullerTest.py

```diff
@@ -0,0 +1,91 @@
+import os.path
```

Comment:
This file should be in the `tests` directory of the repo, not inside the nbgitpuller package